agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -186,6 +187,14 @@ function install_k8s_agent {
     fi
 
     sed -i -e "s|# serviceAccount: sysdig-agent|serviceAccount: sysdig-agent|" /tmp/sysdig-agent-daemonset-v2.yaml
+    # add label for Sysdig instance
+    # -i.bak argument used for compatibility between mac (-i '') and linux (simply -i) 
+    if [ ! -z "$SYSDIG_INSTANCE_NAME" ]; then
+        sed -i.bak -e "7s/^//p; 7s/^.*/    sysdig-instance: $SYSDIG_INSTANCE_NAME/" /tmp/sysdig-agent-daemonset-v2.yaml
```

Comment:
These sed lines assume the contents of the file will not change. It would be safer to add these new labels after each of the lines containing `\(.*\)labels:`

```
sed -i.bak -e 's/\(.*\)labels:/&\
\1  '"sysdig-instance: $SYSDIG_INSTANCE_NAME/" /tmp/sysdig-agent-daemonset-v2.yaml
```

Reply:
I tested that on osx, but it should work in linux